package.json

```diff
@@ -94,8 +94,8 @@
     "serve": "^11",
     "start-server-and-test": "~1.14.0",
     "svelte": "~3.52.0",
-    "tslib": "^2.4.1",
-    "typescript": "~4.9.5",
+    "tslib": "^2.5.3",
+    "typescript": "~5.1.3",
```

Comment:
appreciate it's possible outside the scope of this PR, but I think `tslib` should also be listed as a `peerDep`, because all non-relative imports are treated as external in the rollup config https://github.com/guardian/consent-management-platform/blob/main/src/rollup.config.js#L13